src/Parser/BalanceDruid/Modules/Spells/NewMoon.js

```diff
@@ -67,20 +71,13 @@ class NewMoon extends Module {
   statistic() {
     const abilityTracker = this.owner.modules.abilityTracker;
 
-    const offSet = this.firstMoonTime + 15;
-    const totalFromCD = ((this.owner.fightDuration / 1000) - offSet) / 15;
-    const eachMoon = Math.floor(totalFromCD / 3);
-    this.nmAvailableCasts = eachMoon + 1;
-
-    const extraMoons = ((totalFromCD / 3) - eachMoon) * 3;
-    if (extraMoons > this.newMoonOrder) this.nmAvailableCasts += 1;
-
     const nmCasted = abilityTracker.getAbility(SPELLS.NEW_MOON.id).casts;
+    const nmAvailable = this.getnmAvailableCasts();
```

Comment:
This temp property can be omitted if you turn `getnmAvailableCasts` into a getter